common/reducers/config.js

```diff
@@ -20,38 +19,28 @@ export const initialState: State = {
   gasPriceGwei: 21
 };
 
-function changeLanguage(state: State, action: ChangeLanguageAction): State {
```

Comment:
But why? Cant you just let it sit alone? :) Switch is not scalable, you cant extract it to another file for example, helper variables are awkward because of scope rules, etc